docs/src/examples/lorenz_example.md

```diff
@@ -1,26 +1,289 @@
 
-## The structure of the example script
-First we create the data and the setting for the model
-1. Set up the forward model.
-2. Construct/load the truth data. Store this data conveniently in the `Observations.Observation` object
+The L96 dynamics are solved with RK4 integration.
 
-Then we set up the inverse problem
-3. Define the prior distributions. Use the `ParameterDistribution` object
-4. Decide on which `process` tool you would like to use (we recommend you begin with `Invesion()`). Then initialize this with the relevant constructor
-5. initialize the `EnsembleKalmanProcess` object
+# Structure
 
-Then we solve the inverse problem, in a loop perform the following for as many iterations as required:
-7. Obtain the current parameter ensemble
-8. Transform them from the unbounded computational space to the physical space
-9. call the forward map on the ensemble of parameters, producing an ensemble of measured data
-10. call the `update_ensemble!` function to generate a new parameter ensemble based on the new data
+The example is structured with two distinct components: 1) L96 dynamical system solver; 2) calibrate-emulate sample code. Each of these are described below.
+
+Forward model: The forward mapping from input parameters to output statistics of the L96 system is solved using the `GModel.jl` code, which runs the L96 model across different input parameters ``\theta``. The source code for the L96 system solution is within the `GModel_common.jl` code. 
+
+Calibrate: The main code is located in `calibrate.jl` which provides the functionality to run the L96 dynamical system (within the `GModel.jl` code), extract time-averaged statistics from the L96 states, and use the time-average statistics for calibration.
+
+Emulate-sample: The main code is located in `emulate_sample.jl` which provides the functionality to run the L96 dynamical system, extract time-averaged statistics from the L96 states, and use the time-average statistics for emulation and sampling (uncertainty quantification).
+
+# Inputs
+
+Lorenz dynamics settings: The use of the transient forcing term is with the flag, `dynamics`. Stationary forcing is `dynamics=1` ($A=0$) and transient forcing is used with `dynamics=2` ($A\neq0$).
+The default parameters are specified in `calibrate.jl` and can be modified as necessary.
+The system is solved over time horizon $0$ to `tend` at fixed time step `dt`.
+```julia
+N = 36
+dt = 1/64
+t_start = 800
+```
+
+Inverse problem settings: The states are integrated over time `Ts_days` to construct the time averaged statistics for use by the optimization.
+The specification of the statistics to be gathered from the states are provided by `stats_type`.
+The Ensemble Kalman Process (EKP) settings are
+```julia
+N_ens = 20 # number of ensemble members
+N_iter = 5 # number of EKI iterations
+```
+
+Setting up the Inverse Problem: The goal is to learn ```F_s``` and ```A``` based on the time averaged statistics in a perfect model setting.
```

Comment:
Could we remove all these paragraph starters of `Wxyz: we ... `  , it somewhat impedes the flow just say `We ...`